web/templates/twig-source/incuna-surveys/form/base/proportion-field.html

```diff
@@ -20,11 +12,15 @@
                         >
                     {% block input %}
                         <div class="proportion-input" integer-field model="model[$index]" id="field.id" form="form"></div>
+                        <div
+                            aif-slider-input
+                            model="model[$index]"
+                            ceiling="100"
```

Comment:
That was my first approach, but the slider doesn't accept a width. Limiting the value gives us exactly the same result though. I will need to expose the width for css purposes however